app/models/appeal.rb

```
@@ -297,10 +297,10 @@ def repository
 
     def certify(appeal)
       form8 = Form8.find_by(vacols_id: appeal.vacols_id)
+      certification = Certification.find_by(vacols_id: appeal.vacols_id)
 
       fail "No Form 8 found for appeal being certified" unless form8
```

should we throw an error here too if we can't find the certification